Cargo.toml

```diff
@@ -82,6 +82,9 @@ serial_test = "0.6"
 tempfile = "3.3.0"
 walkdir = "2"
 wat = { version = "1.0" }
+rust_exec_tests = { path = "tests/rust_exec_tests", artifact = "bin", target = "x86_64-unknown-linux-musl" }
+sev_attestation = { path = "tests/sev_attestation", artifact = "bin", target = "x86_64-unknown-linux-musl" }
```

Comment:
I don't see anywhere below using CARGO_BIN_FILE_SEV_ATTESTATION, does this need to be a binary dependency?